### DISCUSSION TITLE
Pin option to pin the frequent used accounts in top! (Ente Auth)

### DISCUSSION THREAD

**Discussion by NeeL21YT** [ente-io/ente#2403] [Enhancements]
Described in the title!
For all devices especially Android and iOS cause the screens are small to navigate with!

**Comment by brog-io** [comment]
Hey there, this is already added. You can swipe to left on a code you want to pin.

**Reply by NeeL21YT** [reply]
Yeah i noticed later when i was installing the app to other devices, maybe i ignored the tutorial first lol
